general.lua

```diff
@@ -215,7 +186,8 @@ local function ScanSpells(event)
 	local changed = false
 	ns.generation = ns.generation + 1
 
-	for tab = 1, 2 do
+	local numTabs = GetNumSpellTabs()
```

Comment:
Not sure how this is on classic, but on live the first two tabs are the ones containing the active spells (1st is general, 2nd is the active spec). Scanning throught the others is just waste.